dsl/properties/scripts/ImportMicroservices.groovy

```diff
@@ -112,7 +112,11 @@ public class ImportMicroservices extends EFClient {
 
         // Service Fields
         efService.service.defaultCapacity = serviceConfig.deploy?.replicas
-        efService.service.minCapacity = serviceConfig.deploy?.updateConfig?.parallelism - serviceConfig.deploy?.replicas
+        if (serviceConfig.deploy?.replicas && serviceConfig.deploy?.updateConfig?.parallelism) {
+            efService.service.minCapacity = serviceConfig.deploy?.replicas - serviceConfig.deploy?.updateConfig?.parallelism
+        } else {
+            efService.service.minCapacity = serviceConfig.deploy?.updateConfig?.parallelism
+        }
```

Comment:
Now with import, the calculation has to be reversed. Given parallelism and replicas we have to count defaultCapacity - minCapacity.
So, default replicas to 1 if not set and same for parallelism.
Then,
defaultCapacity  = replicas
and 
minCapacity = replicas - parallelism